### PR TITLE
Yields fixes

### PR DESCRIPTION
**Pull request by vrtnd**

### CODE REVIEW

File: src/components/Lending/index.tsx [hunk 1]
```diff
@@ -133,24 +137,25 @@ const Lending = ({ data: { yields: initialData, ...props }, isLoading }) => {
...
 			...new Set(
-				poolPairs
-					.map((p) =>
-						[p, p.borrowPool].map(
-							(p) => `${mapChainName(p?.chain?.toLowerCase())}:${p?.underlyingTokens?.[0]?.toLowerCase()}`
-						)
-					)
+				[...lendingPools, ...borrowPools]
+					.map((p) => `${mapChainName(p?.chain?.toLowerCase())}:${p?.underlyingTokens?.[0]?.toLowerCase()}`)
 					.flat()
```
Comment (0xngmi) [inline src/components/Lending/index.tsx:146]: this flat() may be unnecessary?

File: src/components/Lending/index.tsx [hunk 2]
```diff
@@ -249,30 +254,38 @@ const Lending = ({ data: { yields: initialData, ...props }, isLoading }) => {
 							  ]?.price;
 
 					const borrowTokenPrice =
-						selectedBorrowToken === 'STABLES'
+						selectedBorrowToken === 'STABLES' || borrowPool?.category === 'CDP'
```
Comment (0xngmi) [inline src/components/Lending/index.tsx:257]: technically there can be CDPs against other currencies, eg reflexer is a cdp against a coin thats worth ~3.14$

File: src/components/Yields/Filters.tsx [hunk 3]
```diff
@@ -65,6 +75,15 @@ const useAdvancedFilter = (initialData, config = {}) => {
 				filteredData = filteredData.filter((item) => item.symbol.toLowerCase().includes(searchLower));
 			}
 
+			if (includeRewardApy === 'false') {
+				filteredData = filteredData.map((item) => {
+					return {
+						...item,
+						apy: item.apy - (item.apyReward || 0)
```
Comment (0xngmi) [inline src/components/Yields/Filters.tsx:82]: why negative?

File: src/components/Yields/index.tsx [hunk 4]
```diff
@@ -34,7 +39,7 @@ const YieldsRow = ({ data, index, style }) => (
 				alt={data[index].chain}
 			/>
 		</YieldsCell>
-		<YieldsCell>{data[index].apyMean30d.toFixed(2)}%</YieldsCell>
+		<YieldsCell>{data[index].apy.toFixed(2)}%</YieldsCell>
```
Comment (0xngmi) [inline src/components/Yields/index.tsx:42]: why change this?

**Review by 0xngmi — CHANGES_REQUESTED** [review]

File: src/components/Yields/Filters.tsx (hunk 3 above)
Comment (vrtnd) [reply src/components/Yields/Filters.tsx:82]: apy is sum of apyReward + apyBase

File: src/components/Yields/index.tsx (hunk 4 above)
Comment (vrtnd) [reply src/components/Yields/index.tsx:42]: see ^

**Pull request merged**